source/common/upstream/load_balancer_impl.cc

```diff
@@ -67,8 +68,17 @@ void LoadBalancerBase::recalculatePerPriorityState(uint32_t priority) {
                                  host_set.healthyHosts().size() / host_set.hosts().size()));
   }
 
-  // Now that we've updated health for the changed priority level, we need to caculate percentage
+  // Now that we've updated health for the changed priority level, we need to calculate percentage
   // load for all priority levels.
+
+  // The following cases are handled here;
```

Comment:
I think we should update https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/load_balancing.html#panic-threshold?